### ISSUE TITLE
Search is broken on all my KunstmaanBundlesCMS websites

### ISSUE THREAD

**Issue by iknowfoobar** [Kunstmaan/KunstmaanBundlesCMS#843]
I can't be sure when this happened as it has only just been reported by one of our clients. However when we tested it we are now receiving the same error on all our KunstmaanBundleCMS websites that use the search feature.

When you perform a search we get a 500 error from the results page with the following error:

```bash
[Symfony\Component\Debug\Exception\FatalErrorException]                                                   
  Error: Wrong parameters for Exception([string $exception [, long $code [, Exception $previous = NULL]]]) 
```

Is anyone else seeing this? Has a 3rd party package been updated that has made a change?

Composer packages:
ruflin/elastica                      2.3.1
kunstmaan/bundles-cms                3.4.1
symfony/symfony                      v2.7.7 

Elastic search version: 2.1.0
PHP version: PHP 5.5.9-1ubuntu4.13

We are seeing the same problem on older versions of KunstmaanBundlesCMS as well which had previously been working fine

**Comment by iknowfoobar** [comment]
I also get the same error when running either of the CLI commands
```bash
app/console kuma:search:setup
app/console kuma:search:populate full
```

**Comment by kimausloos** [comment]
Hi, this is actually an error in elastica (see https://github.com/ruflin/Elastica/issues/974 ). This probably means you are using elasticsearch 2.0, a temporary fix is to downgrade to 1.7.x, this fixes the exception...

**Comment by iknowfoobar** [comment]
Hi, thanks. Yup going back to 1.7.x fixed everything.
